team/bundles/org.eclipse.jsch.ui/src/org/eclipse/jsch/internal/ui/preference/PreferencePage.java

```diff
@@ -101,6 +108,7 @@ public class PreferencePage extends org.eclipse.jface.preference.PreferencePage
 	private Button ssh2HomeBrowse;
 	Button keyGenerateDSA;
 	Button keyGenerateRSA;
+	private Spinner keyGenerateSize;
```

Comment:
Better name it `keySizeValue` or just `keySize`, but then you have to rename the existing local variable with the same name to `keySizeValue` or alike or you just inline `keySize.getSelection()`
```suggestion
	private Spinner keySizeValue;
```